### PR TITLE
Update .Net Core 2.1 Quickstart

### PR DESCRIPTION
**Pull request by alexisluque**
Code side also updated auth0-samples/auth0-aspnetcore-mvc-samples#47.

### CODE REVIEW

**Comment by albertoperdomo** [comment]
Hello @damieng,
can you please review this PR when you have some time?

File: articles/quickstart/webapp/aspnet-core/_includes/_setup.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 The Callback URL of your application is the URL where Auth0 will redirect to after the user has authenticated in order for the OpenID Connect middleware to complete the authentication process.
 
-You will need to add this URL to the list of Allowed URLs for your application. The Callback URL for the seed project is `http://localhost:60856/signin-auth0` if you use IIS Express, or `http://localhost:5000/signin-auth0` if you use Kestrel, so be sure to add this to the **Allowed Callback URLs** section of your application.
+You will need to add this URL to the list of Allowed URLs for your application. The Callback URL for the seed project is `http://localhost:3000/callback`, so be sure to add this to the **Allowed Callback URLs** section of your application.
```
Comment (damieng) [inline articles/quickstart/webapp/aspnet-core/_includes/_setup.md:5]: What's the rational in changing the URL from `signing-auth0` to `callback`?

Comment (alexisluque) [reply articles/quickstart/webapp/aspnet-core/_includes/_setup.md:5]: The callback URL was changed to be consistent with the other web app quickstarts.

**Review by damieng — APPROVED** [review]

**Comment by cocojoe** [comment]
@alexisluque please check the deployment, make sure it looks as you expect on server. Thx

**Comment by alexisluque** [comment]
@cocojoe it looks like expected.

**Pull request merged**